unityfiles/Assets/Scripts/Game/Weapon/GunController.cs

```diff
@@ -47,10 +46,19 @@ public class GunController : MonoBehaviour {
         set {
             this.fireRate = value;
             this.SetBulletWaitTime(value);
-            this.fireRateTimer.Update(this.timerUID, this.bulletWaitTime);
+            
+            bool suc = this.fireRateTimer.Update(this.timerUID, this.bulletWaitTime);

```

Comment:
Noen grunn for at den settes til en bool ?

Reply:
det ble jort under debug og glemt og fjernet